db/models/reviews.models.js

```diff
@@ -1,5 +1,24 @@
 const db = require("../connection.js");
 
+exports.fetchAllReviews = (sort) => {
+  const queryValues = [],
+    categories = ["euro game", "dexterity", "social deduction", undefined];
+  let queryString = `SELECT reviews.*, COUNT(comments.review_id)::INT AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id `;
+  if (!categories.includes(sort))
+    return Promise.reject({ status: 404, msg: "No such category" });
+  if (sort) {
```

Comment:
I think this variable should be called category as we have named the greenlist of possible values for this variable, "categories"